pkg/background/generate/controller.go

```diff
@@ -122,11 +122,21 @@ func (c *GenerateController) ProcessUR(ur *kyvernov2.UpdateRequest) error {
 		if err != nil {
 			if strings.Contains(err.Error(), doesNotApply) {
 				logger.V(3).Info(fmt.Sprintf("skipping rule %s: %v", rule.Rule, err.Error()))
+			} else {
+				var errForEvent error
+				if strings.Contains(err.Error(), "object has been modified") {
+					errForEvent = fmt.Errorf("failed to generate resource: resource already exists or has been modified")
```

Comment:
Why do we need to check the error message here? Can't we create a generic failure event?